assemble.go

```diff
@@ -8,6 +8,21 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+// InvalidSeedAction represent the action that we will take if a seed
+// happens to be invalid. There are currently two options: either fail with
+// an error or skip the invalid seed and try to continue.
+type InvalidSeedAction int
+
+const (
+	IsaBailOut InvalidSeedAction = iota
+	IsaSkip
+)
+
+type AssembleOptions struct {
+	N   int
+	Isa InvalidSeedAction
```

Comment:
Feel free to rename `Isa` to `InvalidSeedAction` (it won't clash with the type above) as well if that makes things more readable. If you prefer to keep it short, `ISA` would work too as per https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done, I used `InvalidSeedAction`.